src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Cardinality management'
+metaDescription: 'You can manage your data cardinality from the New Relic UI.'
+freshnessValidatedDate: 2024-09-24
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+As an administrator at New Relic, one of your tasks is to make sure you don't go beyond your limit of GB Ingested. One tool for tracking your ingest is the Cardinality Management UI. You'll find helpful graphs and tables explaining your ingest so you can take action if you're approaching limits.
+
+## Where can you manage cardinality? [#where-manage]
+
+In the New Relic UI, you can find this with other administration tools. Click your name in the lower left and go to **Administration > Cardinality Management**.
+
+## Account Cardinality Breakdown [#account-breakdown]
+
+If you're in an organization with multiple accounts, you can see a breakdown of cardinality by each account. As with most capabilities within New Relic, you can change the account you are currently viewing by selecting the account picker in the top right corner. The UI will display information about the selected account, as long as you have access to that account. 
+
+### Usage over time [#usage-over-time]
+
+On the opening page of **Cardinality Management**, you can view the **Usage over time** chart:
+
+<img
+    style={{ align: 'left',maxWidth: '90%' }}
+    title="Screenshot showing the usage over time chart"
+    alt="Screenshot showing the usage over time chart"
+    src="/images/apis-and-data_screenshot-crop_usage-over-time.webp"
+/>
+
+This shows following:
+
+* The limit currently at 5 million account cardinality.
+* The current cardinality usage for the account, which is in the 2 to 4 million range. If there's no change in the value, then a new data point isn't reported. This can appear as an empty data point, but it's not data loss. This behavior may change as the feature is finalized.
+
+
+<Callout variant="tip">
+  You can modify this timeseries by changing the timepicker in the top right corner. Also, the timepicker only affects the timeseries and does not affect the **Metrics** table under the timeseries.
```

Comment:
Is it timepicker or time-picker or time picker? I'm actually not sure

Reply:
@jeff-colucci it looks like "time-picker" is the most common on our site. I'll switch to that!